index.html

```diff
@@ -0,0 +1,143 @@
+        </p>
+        <div class="program-date">
+          <p id="bold-text" class="mobile-font">2023.04.17(THU) ~ 23(WED)</p>
+          <p class="venue grey-text mobile-font">@ Eagle Square, Abuja, Nigeria</p>
+        </div>
+      </section>
+    </header>
+    <section id="program-section" class="sub-section-black">
+      <h2 class="section-header-white">Main Program</h2>
+      <div class="middle-underline flex justify-center">
+        <div class="line"></div>
+      </div>
+      <div class="program-list flex flex-column">
+        <div class="flex">
+          <div class="program-icon-div">
+            <img class="program-icons flex" src="images/program-icon-1.png" alt="">
+          </div>
+          <h3 class="sub-section-header-white flex align-center red-color">Lecture</h3>
+          <p class="program-description sub-section-text-white">Listen to the speakers from various countries talk about the hitory and prospects of African storytelling.</p>
+        </div>
+        <div class="flex">
+          <div class="program-icon-div">
+            <img class="program-icons flex" src="images/program-icon-2.png" alt="">
+          </div>
+          <h3 class="sub-section-header-white flex align-center red-color">Auction</h3>
+          <p class="program-description sub-section-text-white">Appreciate various writing styles from writers across all Africa, organized by the West African Art Gallery, Accra</p>
+        </div>
+        <div class="flex">
+          <div class="program-icon-div">
+            <img class="program-icons flex" src="images/program-icon-3.png" alt="">
+          </div>
+          <h3 class="sub-section-header-white flex align-center red-color">Connect</h3>
+          <p class="program-description sub-section-text-white">Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="flex">
+          <div class="program-icon-div">
+            <img class="program-icons flex" src="images/program-icon-4.png" alt="">
+          </div>
+          <h3 class="sub-section-header-white flex align-center red-color">Workshop</h3>
+          <p class="program-description sub-section-text-white">Try writng your own piece using open source license rather than just watching others do it.</p>
+        </div>
+        <div class="flex">
+          <div class="program-icon-div">
+            <img class="program-icons flex" src="images/program-icon-5.png" alt="">
+          </div>
+          <h3 class="sub-section-header-white flex align-center red-color">Tour</h3>
+          <p class="program-description sub-section-text-white">From the Famous Lagos up north to Kaduna and then southwards to Port Harcout, take your time to enjoy our country</p>
+        </div>
+      </div>
+      <div class="join-btn flex">
+        <p class="text-center">Join FicFest 2023</p>
+      </div>
```

Comment:
Kindly adjust the `Layout` of the content of this button to match the template. 👀🛠

Your design
![image](https://user-images.githubusercontent.com/78251927/233441786-081675ed-3df8-4d41-8a0b-edd12d9d094b.png)


Template
![image](https://user-images.githubusercontent.com/78251927/233441675-96bb8fac-8d73-495f-b684-74f0cfed7d1a.png)